src/parsers/buddyinfo.cpp

```diff
@@ -66,7 +67,15 @@ zone parse_buddyinfo_line(const std::string& line)
 
         zn.name = tokens[ZONE_NAME];
 
-        for (size_t i = 0; i < zn.chunks.size(); ++i)
+        // std::distance can be negative, but the conditional on line 46
+        // and the nature of the begin() and end() iterators means num_chunks
+        // should never be less than 0.
+        size_t num_chunks =
```

Comment:
`std::distance` is an overkill here.
As we're using a vector, can we just use `tokens.size() - FIRST_CHUNK`?

std::distance is both less readable (not a very popular method) and less efficient (linear complexity).